server/migrations/1638255797809-SetLoadingStateToFalseForExistingTables.ts

```diff
@@ -0,0 +1,46 @@
+import { AppVersion } from '../src/entities/app_version.entity';
```

Comment:
Will have to update for the template definitions as well

Reply:
👍 

Added. them in this commit : https://github.com/ToolJet/ToolJet/pull/1446/commits/09978dda850256badebfe73de98aee9cc193d3b4